packages/dd-trace/src/llmobs/index.js

```diff
@@ -94,11 +94,15 @@ function handleLLMObsParentIdInjection ({ carrier }) {
 }
 
 function handleFlush () {
+  let err = ''
   try {
     spanWriter.flush()
     evalWriter.flush()
   } catch (e) {
+    err = 'writer_flush_error'
     log.warn(`Failed to flush LLMObs spans and evaluation metrics: ${e.message}`)
+  } finally {
+    telemetry.recordUserFlush(err)
   }
```

Comment:
```suggestion
  } 
  
  telemetry.recordUserFlush(err)
```
shouldn't need the finally here i think since we swallow the error anyways